File: fix_syntax.js
const fs = require('fs');
const file = 'src/features/sustainability/pages/EmissionsModule.tsx';
let txt = fs.readFileSync(file, 'utf8');

txt = txt.replace(/\(acc\[\DQS \\\$\\{curr\.dqs\}\\\\]\)/g, "acc[\DQS \\]");
txt = txt.replace(/\(acc\[\DQS \\\$\\{curr\.dqs\}\\)/g, "acc[\DQS \\]");

fs.writeFileSync(file, txt);
console.log('fixed');

File: fix_syntax.py
with open("src/features/sustainability/pages/EmissionsModule.tsx", "r", encoding="utf8") as f:
    t = f.read()

t = t.replace("(acc[`DQS ${curr.dqs}`) || 0) + 1", "(acc[`DQS ${curr.dqs}`] || 0) + 1")

with open("src/features/sustainability/pages/EmissionsModule.tsx", "w", encoding="utf8") as f:
    f.write(t)
print("done")
